packages/core/src/editor/BlockNoteTipTapEditor.ts

```diff
@@ -184,7 +184,10 @@ export class BlockNoteTipTapEditor extends TiptapEditor {
 
     // emit the created event, call here manually because we blocked the default call in the constructor
     // (https://github.com/ueberdosis/tiptap/blob/45bac803283446795ad1b03f43d3746fa54a68ff/packages/core/src/Editor.ts#L117)
-    this.commands.focus(this.options.autofocus);
+    this.commands.focus(
+      this.options.autofocus ||
+        this.options.element.getAttribute("data-bn-autofocus") === "true"
```

Comment:
This seems strange, I've always found the autofocus behavior strange in tiptap shouldn't we just use the normal HTML version and read that from the element?

Reply:
I played around with this, but somehow `element.autofocus` returns false even when I set it to true on the div. Maybe sth with browser contenteditable compatibility.

The good thing however is that for the user our API does feel more HTML-like (they can do `<BlockNoteView autofocus={true}>`)